proxy/backend/core/src/main/java/org/apache/shardingsphere/proxy/backend/response/data/QueryResponseCell.java

```diff
@@ -18,16 +18,26 @@
 package org.apache.shardingsphere.proxy.backend.response.data;
 
 import lombok.Getter;
-import lombok.RequiredArgsConstructor;
 
 /**
  * Query response cell.
  */
-@RequiredArgsConstructor
 @Getter
 public final class QueryResponseCell {
     
     private final int jdbcType;
     
     private final Object data;
+    
+    private String columnTypeName;
```

Comment:
If this field is nullable, is it possible to make it final and add an Optional getter?

Reply:
ok,